releasenotes/notes/dag-deprecate-none-args-e6ef868148da275d.yaml

```
@@ -0,0 +1,7 @@
+---
+deprecations:
```

Because this is technically a breaking change I would put it in the `upgrade` section:
```suggestion
upgrade:
```